pwiz_tools/Skyline/app.config

```diff
@@ -771,6 +771,9 @@
             <setting name="SyncMZScale" serializeAs="String">
                 <value>False</value>
             </setting>
+            <setting name="FullScanPeakType" serializeAs="String">

```

Comment:
I left it as Peak Type because Scan Type is already taken for MS1/MS2 selection. Let me know if you have any other suggestions for the name.